gdx-pay-iosrobovm-apple/src/main/java/com/badlogic/gdx/pay/ios/apple/PurchaseManageriOSApple.java

```diff
@@ -534,15 +536,31 @@ public Information getInformation(String identifier) {
                         .priceCurrencyCode(p.getPriceLocale().getCurrencyCode())
                         .priceInCents(MathUtils.ceilPositive(p.getPrice().floatValue() * 100))
                         .priceAsDouble(p.getPrice().doubleValue())
+                        .freeTrialPeriod(freeTrialPeriod(p))
                         .build();
                 }
             }
         }
         return Information.UNAVAILABLE;
     }
 
+    private FreeTrialPeriod freeTrialPeriod(SKProduct product) {
```

Comment:
better name was `parseFreeTrial`

Reply:
I think we can name it better (in both Google Billing and iOS implementation) convertToFreeTrialPeriod? The SKProduct has the fields, so we are not parsing a String like we do in Google Billing impl.

Reply:
That's good, too!